src/web/app/src/components/DependenciesTable/Row.tsx

```diff
@@ -36,6 +38,9 @@ const useStyles = makeStyles((theme) => ({
     color: theme.palette.text.secondary,
     fontSize: '2rem !important',
   },
+  dependencyInfoCell: {
+    borderBottomColor: '#909090',
```

Comment:
So this hardcoded color gonna be for dark and light mode?

Reply:
Also not sure if okay about this color difference  @humphd 
![image](https://user-images.githubusercontent.com/58233223/195235297-32bdd0e9-9b42-4e6a-a55c-51dc4be3e055.png)

Reply:
> So this hardcoded color gonna be for dark and light mode?

Yea, it would be the same for both dark and light mode. Should this be changed to only effect dark mode?

Reply:
That color difference was not intentional. It should be the same color. I will look into why that is happening

Reply:
This has been fixed by setting the border width in em instead of px.